### PR TITLE
Display Java version and home in bloop about

### PR DESCRIPTION
**Pull request by cquiroz**
As suggested in gitter, this PR changes `bloop about` to show the JVM version and java home bloop is running on:

image

### CODE REVIEW

File: frontend/src/main/scala/bloop/engine/Interpreter.scala [hunk 1]
```diff
@@ -98,10 +98,16 @@ object Interpreter {
     val scalaVersion = bloop.internal.build.BuildInfo.scalaVersion
     val zincVersion = bloop.internal.build.BuildInfo.zincVersion
     val developers = bloop.internal.build.BuildInfo.developers.mkString(", ")
+    val javaVersion = Option(System.getProperty("java.version"))
```
Comment (tgodzik) [inline frontend/src/main/scala/bloop/engine/Interpreter.scala]:
```scala
sys.props.get("java.version")
sys.props.get("java.home")
```
Should get an option and noe need to pack it in one here.

Comment (jvican) [reply frontend/src/main/scala/bloop/engine/Interpreter.scala]: I actually prefer this over using the Scala API :smile:

Comment (tgodzik) [reply frontend/src/main/scala/bloop/engine/Interpreter.scala]: Haha, fine with me. Just thought it might be a bit shorter.

**Comment by cquiroz** [comment]
Thanks. I didn't think you'd check this on test but great that you do. I'll try to fix the test today

**Pull request merged**